week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(pairs){
+   if (Array.isArray(pairs) && pairs.length==2){
+    return pairs;
```

Comment:
This will technically work, but most of the time with `.filter()` we want to return a boolean. Here we're returning the element of the array instead. How could we fix this?